README.md

```diff
@@ -1,31 +1,56 @@
 
 # lm-logs-fluentd (beta)
-This plugin sends Fluentd records to the Logic Monitor
+This output plugin sends Fluentd records to the configured LogicMonitor account.
 
-## Getting started
-- Add `out_lm.rb` to plugins directory
+## Prerequisites
 
-- Add to `fluent.conf`
+Add `out_lm.rb` to your Fluentd plugins directory.
+
+## Configure the output plugin
+
+Create a custom `fluent.conf` or edit the existing one to specify which logs should be forwarded to LM.
 
-Minimal configuration:
 ```
+# Match events tagged with "lm.**" and
+# send them to LogicMonitor
 <match lm.**>
-    @type lm
-    company_name {company_name}
-    resource_mapping {"event_key": "lm_property"}
-    access_id {lm_access_id}
-    access_key {lm_access_key}
-    flush_interval 1s
-    debug false
+@type lm
+company_name <your_company_name>
+resource_mapping {"<event_key>": "<lm_property>"}
+access_id <your_lm_access_id>
+access_key <your_lm_access_key>
+
```

Comment:
Is this space intentional?  And adding tab before the arguments would be better.

Reply:
it was, but I guess there's no reason for it. Removed the space and added tabs.